Web/starter_project_vanguard/types/teams.ts

```diff
@@ -0,0 +1,9 @@
+type SocialMedia = { url: string; social: string }
+
+export type Member = {
```

Comment:
[very-nit] we can rename this to `TeamMember` to stay consistent with other teams if we ever do another PR